CHANGELOG.md

```diff
@@ -135,6 +135,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fail weight update when decommission ongoing and fail decommission when attribute not weighed away ([#4839](https://github.com/opensearch-project/OpenSearch/pull/4839))
 - Skip SymbolicLinkPreservingTarIT when running on Windows ([#5023](https://github.com/opensearch-project/OpenSearch/pull/5023))
 - Change the output error message back to use OpenSearchException in the cause chain. ([#5081](https://github.com/opensearch-project/OpenSearch/pull/5081))
+- Allow mmap to use new preview JDK-19 APIs in Apache Lucene 9.4+ ([#5151](https://github.com/opensearch-project/OpenSearch/pull/5151))
```

Comment:
We just pushed an update to the [changelog policy](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog) that applies the learnings from the recent 2.4 release.

This is probably an example that can skip the changelog? Whatever changes make use of the new APIs would be relevant, but this is probably too low level.

Reply:
Thanks @andrross , I could be mistaken but it falls into `A newly added feature`: we activate a new Apache Lucene feature by changing the JVM parameters, what do you think from this perspective? (I am fine removing changelog line as well)

Reply:
Ah, does Lucene detect this setting and change its behavior at runtime accordingly? If so then you're right :)

Reply:
Thanks @andrross !